tests/test_scrape.py

```diff
@@ -71,5 +72,5 @@ def test_data():
         json=sample_data
     )
 
-    prom_data = data(enrollment, token, month='2017-03')
-    assert prom_data.count('azure_costs') == 4
+    prom_data = data(enrollment, token, metric_name, month='2017-03')
+    assert prom_data.count(metric_name) == 4
```

Comment:
1) Since extracting the costs is _the_ purpose of your tool, I would love to see a test that checks the costs are reported correctly given the sample response you specified.
2) The solution you have surely is simpler, but without the explanation you gave in the comments it has the air of magic ;-). This could be fixed like this:

    occurrences_of_metric_name_per_record = 2  # including metrics description
    number_of_records = len(sample_data)
    assert prom_data.count(metric_name) == number_of_records * occurrences_of_metric_name_per_record

This solution would have your explanation in code. Alternatively, you could add a larger comment.